Automattic/woocommerce-payments
# Simplify deposits UI - Add billing schedule section heading

Fixes #5699 

#### Changes proposed in this Pull Request

This PR adds the section heading before the deposits history table. This section includes details about the current deposit schedule:

### Daily

<img width="400" alt="Screenshot 2023-03-29 at 11 26 00 am" src="https://user-images.githubusercontent.com/8490476/228402636-0739c924-3282-44b4-9a79-bf309d20b40c.png">


### Weekly

| Tuesday        | Friday           |
| ------------- |:-------------:|
| <img width="700" alt="Screenshot 2023-03-29 at 11 27 23 am" src="https://user-images.githubusercontent.com/8490476/228402784-35c09dee-f384-4106-ae3b-033017ed9b83.png">      | <img width="697" alt="Screenshot 2023-03-29 at 11 27 57 am" src="https://user-images.githubusercontent.com/8490476/228402844-f24776d2-13ce-4dd5-9988-b12384e5be6b.png"> | 


### Monthly

| Date (6th)        | Last day           |
| ------------- |:-------------:|
| <img width="697" alt="Screenshot 2023-03-29 at 11 28 19 am" src="https://user-images.githubusercontent.com/8490476/228402916-2df8fd6c-d893-4961-afed-c6a59be352ce.png">      | <img width="703" alt="Screenshot 2023-03-29 at 11 29 32 am" src="https://user-images.githubusercontent.com/8490476/228403073-19a89ba4-1b76-46c3-bc55-5be3cc92da99.png"> | 

### Suspended

<img width="400" alt="Screenshot 2023-03-30 at 5 07 42 pm" src="https://user-images.githubusercontent.com/8490476/228756749-6e856b99-7c26-43dd-b8c7-547891138120.png">


#### Testing instructions





1. Set the wp option `_wcpay_feature_simplify_deposits_ui` to `1` to enable the feature flag.
2. Go to the **Payments > Overview** page
3. Note the Deposit History section heading and deposit schedule.
4. Click the link to change your deposit schedule in the footer. 
5. Change the schedule to any combination and confirm the deposit schedule updates in the deposit history section. 

To test the blocked status

> **Note**
> Changes introduced in https://github.com/Automattic/woocommerce-payments/pull/5864 are necessary to prevent an error if you use the Server approach below. 

1. Disable your WC Payments accounts. To do this there are 2 options.
    1. On client. Change the `depositStatus` in [`get_account_status_data()](https://github.com/automattic/woocommerce-payments/blob/issue/5699-display-billing-schedule/includes/class-wc-payments-account.php#L220)` so `'blocked'` is returned. 
    2. On Server change the `is_deposits_blocked()` function to always return `true`.
2. Refresh the **Payments > Overview** page and confirm the deposit schedule section now reads as: "Your deposits are temporarily suspended ([learn more](https://woocommerce.com/document/payments/faq/deposits-suspended/))"

-------------------


**Post merge**
